src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
     }
 };
 
+groupTypes.horizBrace = function(group, options) {
+    const style = options.style;
+    const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+        style.sizeMultiplier;
+
+    const haveNote = (group.type === "supsub");
+    let note;
+    let newOptions;
+    let noteReset;
+    if (haveNote) {
+        if (group.value.sup) {
+            newOptions = options.withStyle(style.sup());
+            note = buildGroup(group.value.sup, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sup().cls()],
+                [note], newOptions);
+        } else {
+            newOptions = options.withStyle(style.sub());
+            note = buildGroup(group.value.sub, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sub().cls()],
+                [note], newOptions);
+        }
+        group = group.value.base;
+    }
+
+    // Build the base group
+    const body = buildGroup(
+       group.value.base, options.withStyle(style.cramp()));
+
+    // If a custom bracket, convert units.
+    if (group.value.thickness) {
+        group.value.thickness.value = calculateSize(group.value.thickness.value,
+                options.style);
+    }
+    if (group.value.height) {
+        group.value.height = calculateSize(group.value.height.value,
+                options.style);
+    }
+
+    // Create the stretchy element
+    const braceBody = stretchy.svgSpan(group, options);
+
+    // Generate the vlist, with the appropriate kerns
+    const braceShift = (group.value.isOver ? -body.height - 3 * ruleWidth :
+                        body.depth + braceBody.height + 3 * ruleWidth);
+
+    let vlist = buildCommon.makeVList([
+        {type: "elem", elem: body, shift: 0},
+        {type: "elem", elem: braceBody, shift: braceShift},
+    ], "individualShift", null, options);
```

Comment:
Thanks for the awesome explanation.

I didn't see that `vlist` gets used by `vspan` when `haveNote` is `true`.  This is fine as is.  

Using `firstBaseline` vs `individualShift` isn't a big deal.  I was curious more than anything else.